lib/options.js

```diff
@@ -332,8 +332,8 @@ function parseUnsuppress(args) {
 }
 
 /**
- * @param {string | string[] | *} input
```

Comment:
I think that would also be a valid typing.

Reply:
I'd say so (but can happen in a later PR if you prefer), because I'm pretty sure that at the moment the input is sometimes `undefined`, so it seems that TS doesn't catch that (probably because they're `any`?)

Reply:
Yeah, that would make sense. I can fix it.